docs/install.rst

```diff
@@ -113,16 +113,20 @@ Installing Rally
 ----------------
 
 1. Ensure ``~/.local/bin`` is in your ``$PATH``.
-2. Install Rally: ``python3 -m pip install --user esrally``.
+2. Ensure pip is the latest version: ``python3 -m pip install --user --upgrade pip``
+3. Install Rally: ``python3 -m pip install --user esrally``.
 
 VirtualEnv Install
 ------------------
 
 You can also use Virtualenv to install Rally into an isolated Python environment without sudo.
 
-1. Set up a new virtualenv environment in a directory with ``python3 -m venv .``
-2. Activate the environment with ``source /path/to/virtualenv/dir/bin/activate``
-3. Install Rally with ``python3 -m pip install esrally``
+1. Set up a new virtualenv environment in a directory with ``python3 -m venv .venv``
```

Comment:
One thing to note here that is different to the `--user` case is that even with e.g. python 3.8.7 with comes with `pip 21.0.1`, creating a `venv` brings pip `20.2.3`. Starting with Python `3.9.0`, there's a new flag [--upgrade-deps](https://docs.python.org/3.9/library/venv.html) that can do this in one go.